### ISSUE TITLE
Print version exactly as it is

### ISSUE THREAD

**Issue by tiukhtinvladimir** [fail2ban/fail2ban#2122]
Hi

It is very hard to parse this output in automated systems
```
fail2ban-server --version
Fail2Ban v0.10.3.fix1

Copyright (c) 2004-2008 Cyril Jaquier, 2008- Fail2Ban Contributors
Copyright of modifications held by their respective authors.
Licensed under the GNU General Public License v2 (GPL).
```
It will be nice to have something like
```
fail2ban-server --version-short
0.10.3.1
```
Thanks

**Comment by sebres** [comment]
Hmm... I don't see the necessity to put this "logo" information at all by the usage of `--version`.
Possibly for backwards compatibility purposes only.

So instead of `--version-short`...
@fail2ban/contributors, @fail2ban/maintainers What would you say if we'll rewrite handling of `--version` to avoid print of the logo-information at all (let do it only by using `--help`, etc.)?
Any objections?
@tiukhtinvladimir as topic-starter: Would it affect you if we'll remove this info by `--version`?

Alternative to `--version-short`:
new option `--nologo` to avoid it on demand.

**Comment by szepeviktor** [comment]
OK. Go ahead.

**Comment by sebres** [comment]
> Ideally I would like to get 0.10.3.1 as an output example.

Of course.... The question was rather: must it be totally backwards compatible just using old option `--version`?

**Comment by tiukhtinvladimir** [comment]
@sebres If I got your question correctly - what ever you think is better for the project. It might be '--version' or any other possible words. But if you ask my opinion I would say that using '--version' is something pretty standardized and clear for anybody. Thanks

**Comment by sebres** [comment]
So implemented now for both major branches, see https://github.com/fail2ban/fail2ban/blob/61e30949b3e4888a24bfcea9881a0450f7428182/ChangeLog#L43-L45

```bash
$ fail2ban-client --version
Fail2Ban v0.10.4.dev1
$ fail2ban-client -V
0.10.4.1
```
